helsingborg-stad/aterbruket-webapp
# Feature/add purchase price

**Explain the changes you’ve made>**
Added a new field in the form for the purchased price.
 
**Explain why these changes are made**
New feature to create more value for Helsingborg. In the Future we can see how much the city is saving. 
 
**Explain your solution**
Updated the backend with a new value. 
added a input in the form for the new value.
Only render the value in itemDetails page if the value is not empty.
 
**How to test the changes?**
_Concrete example:_
 

1. Empty your AWS
2. Create a new add
3. see how it´s rendered in the detail page.
4. change the add with a new value.
